src/cli.yaml

```diff
@@ -120,11 +155,32 @@ subcommands:
 
         Example:
         rbt vcf-baf < calls.bcf > annotated.bcf
+        {n}
```

Comment:
What is this?

Reply:
A line break (claps flavor of `\n`). This was the only option I found to force a new line at the end of the `about` text so that the descriptions for the different subcommands are easier to tell apart.